internal/embedded/embedded.go

```diff
@@ -7,7 +7,20 @@ import (
 //go:embed favicon.ico
 var favicon []byte
 
+//go:embed normalize.css
+var normalize []byte
+
 // Favicon returns the favicon data
 func Favicon() []byte {
 	return favicon
 }
+
+// NormalizeCss returns the normalize css data
+func NormalizeCss() []byte {
+	return normalize
+}
+
+// EmptyCss reset the default css data
+func EmptyCss() []byte {
+	return []byte("/* No Default CSS Loaded */")
+}
```

Comment:
It'd be better if the view didn't include the `<link />` if a `public/default.css` isn't present since that would avoid a roundtrip for this CSS sheet. 

Then we can also remove this function.

Reply:
Done